### PR TITLE
Add pausetimes to nightly runs

### PR DESCRIPTION
**Pull request by Firobe**
This add pausetimes benchmarking on sequential programs in the nightly navajo and turing runs.
It also refactors `run_all_custom.sh` to be a bit more readable.

Tested locally on Navajo

### CODE REVIEW

File: run_all_custom.sh [hunk 1]
```diff
@@ -65,34 +65,49 @@ fi
 COUNT=`jq '. | length' "${CUSTOM_FILE}"`
 
 # Iterate through each variant
-i=0
-while [ $i -lt ${COUNT} ]; do
-    j=0
-    while [ $j -lt 3 ]; do
+for i in $(seq 0 $((${COUNT} - 1))); do
+    # Iterate through each kind of benchmark
+    for KIND in "sequential" "parallel" "perfstat" "pausetimes"; do
```
Comment (punchagan) [inline run_all_custom.sh:70]: Thank you for cleaning this up! I've wanted to this for a while. (did a part of it in #376 which didn't get merged).

**Review by punchagan — COMMENTED** [review]
Should we run `olly latency` for some benchmarks in CI, as well?

**Comment by Firobe** [comment]
Added pausetimes on parallel benchmarks as well, since we want to report both

**Comment by shakthimaan** [comment]
LGTM!

**Pull request merged**